menpofit/unified/algorithm.py

```diff
@@ -0,0 +1,395 @@
+import numpy as np
+
+from menpofit.base import build_grid
+from menpofit.checks import check_model
+from menpofit.result import ParametricIterativeResult
+from menpofit.modelinstance import OrthoPDM
+
+multivariate_normal = None  # expensive, from scipy.stats
+
+
+# Abstract Interface for AAM Algorithms ---------------------------------------
+
+class UnifiedAlgorithm(object):
+
+    r"""
+    Base interface for optimization of UnifiedAAMCLM
+
+    Parameters
+    ----------
+    aam_interface : : `subclass` of :map:`LucasKanadeBaseInterface`, 
+        Concrete instantiation of an interface for Lucas-Kanade optimization of
+        standard AAMs.
+    appearance_model : `menpo.model.PCAModel` or subclass
```

Comment:
The doc needs to be updated here. `appearance_model` and `transform` are not existent.